Header header
string[] joint_names
dish_size_msg_points[] points